src/components/ApiService/ApiService.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios'
+
+const urlport = `${process.env.REACT_APP_BASEURL}:${process.env.REACT_APP_PORT}`
+
+export const getAll = (params,query) => {
```

Comment:
this method could be named just get(), it is more logical because sometimes it will not get "all" there is :)